apps/src/templates/manageStudents/ManageStudentsActionsCell.jsx

```diff
@@ -56,16 +90,45 @@ class ManageStudentActionsCell extends Component {
             <Button onClick={this.onCancel} color={Button.ButtonColor.blue} text="Cancel" />
           </div>
         }
+        <BaseDialog
+          useUpdatedStyles
+          uncloseable
+          isOpen={this.state.deleting}
+          style={{paddingLeft: 20, paddingRight: 20, paddingBottom: 20}}
+        >
+          <h2 style={styles.heading}>Remove student</h2>
+          <div>Are you sure you want to remove this student?</div>
+          <DialogFooter>
+            <Button
+              class="ui-test-cancel-delete"
```

Comment:
I'd been using `uitest` rather than `ui-test`. It does seem like there are a few usages of the latter, but the former is much more common in our codebase.

Reply:
Do these class names get used? Also, React should be warning you that it's className not class, shouldnt it?

Reply:
It doesn't get used, so I'll remove it, but I got no warning :(